raiden-dapp/src/views/GeneralHome.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="general-screen-home">
+    <v-row class="general-screen-home__identicon" no-gutters>
+      <header-identicon></header-identicon>
```

Comment:
Not sure if have (should have? @kelsos) a linting rule for this, but might make sense to make tags without children self closing: `<header-identicon />`